config/thoth.yaml

```diff
@@ -664,3 +664,21 @@ repositories:
       - name: update
         configuration:
           labels: [bot]
+
+  - slug: thoth-station/s2i-generic-data-science-notebook
+    token: "{KEBECHET_TOKEN}"
+    managers:
+      - name: info
+      - name: version
+        configuration:
+          maintainers:
+            - sesheta
+            - harshad16
+            - goern
+          assignees:
+            - sesheta
+          labels: [bot]
+          changelog_file: true
+      - name: update
+        configuration:
+          labels: [bot]
```

Comment:
TIL I remember, this part was used in old days when kebechet used to run as a cronjob.

Reply:
> Any repo is included to kebechet, by installing kebhut app and including `.thoth.yaml` in the repo. There are no changes required in the kebechet repo.

So if we don't need this to install with kebechet I should just close the whole PR correct?

Reply:
yes this change is not needed, there seems to be another change in this PR.
if that is also not needed, this can be closed.